x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,52 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `x/subscription` stores the plans and subscriptions on chain:
+
+```golang
+type Plan struct {
+  id int,  // auto-increasing unique identifier
+  title string,
+  description string,
+  owner Address,  // beneficial owner of the plan
+  price Coin,  // price to pay for each period, Coin contains both amount and denomination
+  expiration_time int,  // the timestamp in seconds when plan expires, it will be removed automatically
+  tolerate_failure bool, // if false, it'll automatically cancel subscriptions fails to pay, otherwise merchant can handle them manually
+
+  interval int, // payment interval defined in number of seconds
+  interval_offset int, // interval offset in terms of seconds to support timezones or any arbitrary time offsets
```

Comment:
Can you share the use case of using `interval_offset` to help me better understanding this? Thanks